### PR TITLE
S1_US5_carousel

### PR DESCRIPTION
**Pull request by Mcbess-san**

### CODE REVIEW

**Review by Vannou28 — APPROVED** [review]

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
why are there changes to the composer.lock file?

File: src/View/Home/index.html.twig [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {% extends 'layout.html.twig' %}
 
 {% block content %}
-    <h1>Nathalie Milliet</h1>
+    <h1> Témoignage </h1>
```
Comment (RaphaelBS-WCS) [inline src/View/Home/index.html.twig:4]: i thought that your section title must not be on the index but on your own view sheet, even if it includes your caroussel.

**Review by RaphaelBS-WCS — COMMENTED** [review]
I left you some comments

**Review by Vannou28 — COMMENTED** [review]
Hi Roman, I have the impression that you deleted the index, is that normal ??? because for me you must have a link in your src / View / Home / testimony.html.twig

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved!

**Review by Vannou28 — APPROVED** [review]

File: src/View/Home/_carousel.html.twig [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<h2> Témoignage </h2>
```
Comment (LucyMerlier) [inline src/View/Home/_carousel.html.twig:1]: spaces before and after text? "témoignage" singular

File: src/View/Home/_carousel.html.twig [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    <button class="carousel-control-prev" type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide="prev">
+        <span class="carousel-control-prev-icon" aria-hidden="true"></span>
+        <span class="visually-hidden">Previous</span>
+    </button>
+    <button class="carousel-control-next" type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide="next">
+        <span class="carousel-control-next-icon" aria-hidden="true"></span>
+        <span class="visually-hidden">Next</span>
+    </button>
+    </div>
+</div>
```
Comment (LucyMerlier) [inline src/View/Home/_carousel.html.twig:46]: indentation

File: src/View/Home/_carousel.html.twig [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+             et je la remercie pour les réponses qu'elle a pu me donner ! "</p>
+            <h3>Océane.H</h3>
+        </div>
+        </div>
+        <div class="carousel-item">
+        <img src="/assets/images/592367.jpg" class="d-block w-100" alt="...">
+        <div class="carousel-caption d-none d-sm-block">
+            <p>" J'ai consulté Mme MILLIET 2 fois en quatre ans et je ne peux que vous la recommander, elle est bienveillante, vous met à l'aise, 
+            vous explique parfaitement bien comment va se dérouler la séance et on ressort de chez elle véritablement apaisée. Pour ma part en tout cas, 
+            je suis ravie et toutes ces prédictions et avertissements se sont avérées utiles et justes. "</p>
```
Comment (LucyMerlier) [inline src/View/Home/_carousel.html.twig:23]: spaces before and after text?

File: src/View/Home/_carousel.html.twig [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    <div class="carousel-indicators">
+        <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
+        <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="1" aria-label="Slide 2"></button>
+        <button type="button" data-bs-target="#carouselExampleCaptions" data-bs-slide-to="2" aria-label="Slide 3"></button>
+    </div>
+    <div class="carousel-inner">
+        <div class="carousel-item active">
+        <img src="/assets/images/592367.jpg" class="d-block w-100" alt="...">
+        <div class="carousel-caption d-none d-sm-block">
+            <p>" Je recommande Nathalie, qui explique très bien comment elle fonctionne,
```
Comment (LucyMerlier) [inline src/View/Home/_carousel.html.twig:13]: hint : there are semantic ways to display quotes in html

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

File: src/View/Home/_carousel.html.twig (hunk 4 above)
Comment (Mcbess-san) [reply src/View/Home/_carousel.html.twig:23]: yes because i would like too have this présenttion for the text

**Review by RaphaelBS-WCS — APPROVED** [review]
Approved

**Review by khadimb — APPROVED** [review]
yes

File: src/View/Home/_carousel.html.twig [hunk 6]
```diff
@@ -0,0 +1,41 @@
+<section id= "testimony" class= "testimony">
+    <h2>Témoignages</h2>
+    <div id="carousel" class="carousel slide" data-bs-ride="carousel">
+    <div class="carousel-indicators">
```
Comment (LucyMerlier) [inline src/View/Home/_carousel.html.twig:4]: indentation inside div

File: src/View/Home/_carousel.html.twig [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    </div>
+    <div class="carousel-inner">
+        <div class="carousel-item active">
+        <img src="/assets/images/592367.jpg" class="d-block w-100 h-100" alt="...">
+        <div class="carousel-caption d-flex justify-content-center">
+            <h3>Océane.H</h3>
+            <p class="d-flex justify-content-center">Je recommande Nathalie, qui explique très bien comment elle fonctionne,
+             et je la remercie pour les réponses qu'elle a pu me donner !
+            </p>            
+        </div>
```
Comment (LucyMerlier) [inline src/View/Home/_carousel.html.twig:17]: indentation inside div

File: src/View/Home/_carousel.html.twig [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+        <div class="carousel-item">
+        <img src="/assets/images/592367.jpg" class="d-block w-100 h-100" alt="...">
+        <div class="carousel-caption d-flex justify-content-center">
+            <h3>Clemence.D</h3>
+            <p class="d-flex justify-content-center">J'ai consulté Mme MILLIET 2 fois en quatre ans et je ne peux que vous la recommander, elle est bienveillante, vous met à l'aise, 
+                vous explique parfaitement bien comment va se dérouler la séance et on ressort de chez elle véritablement apaisée. Pour ma part en tout cas, 
+                je suis ravie et toutes ces prédictions et avertissements se sont avérées utiles et justes.
+            </p>
+        </div>
```
Comment (LucyMerlier) [inline src/View/Home/_carousel.html.twig:27]: indentation inside div

File: src/View/Home/_carousel.html.twig [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+        <div class="carousel-item">
+        <img src="/assets/images/592367.jpg" class="d-block w-100 h-100" alt="...">
+        <div class="carousel-caption d-flex justify-content-center">
+            <h3>Claire.B</h3>
+            <p class="d-flex justify-content-center">Une question quant à votre présent et/ou votre avenir ? Je vous recommande Nathalie. 
+                Tout simplement stupéfiante ! Bienveillante, pédagogue, sensible, souriante et pertinente ! 
+                En 1h, le doute à fait place à l’évidence... Tout simplement : merci d’être sur mon chemin ! À bientôt...
+            </p>                      
+        </div>
```
Comment (LucyMerlier) [inline src/View/Home/_carousel.html.twig:37]: indentation inside div

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
indentation, see comments

**Review by RaphaelBS-WCS — DISMISSED** [review]
I approve

File: src/View/Home/_carousel.html.twig [hunk 10]
```diff
@@ -0,0 +1,41 @@
+<section id= "testimony" class= "testimony">
+    <h2>Témoignages</h2>
+    <div id="carousel" class="carousel slide" data-bs-ride="carousel">
+            <div class="carousel-indicators">
+                <button type="button" data-bs-target="#carousel" data-bs-slide-to="0" class="active" aria-current="true" aria-label="Slide 1"></button>
+                <button type="button" data-bs-target="#carousel" data-bs-slide-to="1" aria-label="Slide 2"></button>
+                <button type="button" data-bs-target="#carousel" data-bs-slide-to="2" aria-label="Slide 3"></button>
+            </div>
```
Comment (LucyMerlier) [inline src/View/Home/_carousel.html.twig:8]: too much indentation

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

**Review by RaphaelBS-WCS — DISMISSED** [review]
Approved

**Review by khadimb — APPROVED** [review]
ok

**Review by LucyMerlier — DISMISSED** [review]

**Review by Vannou28 — APPROVED** [review]
it's good

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**